Missing Express types

Hi,
I'm hitting a transpile error when using the routing npm package without the other fhir-works components:
```
node_modules/fhir-works-on-aws-routing/lib/app.d.ts:2:119 - error TS2307: Cannot find module 'express-serve-static-core' or its corresponding type declarations.

2 export declare function generateServerlessRouter(fhirConfig: FhirConfig, supportedGenericResources: string[]): import("express-serve-static-core").Express; 
```
Just need to add `@types/express-serve-static-core` into the dependencies.
Thanks!
Allan

--- a/package.json
+++ b/package.json
@@ -37,7 +37,8 @@
     "lodash": "^4.17.15",
     "mime-types": "^2.1.26",
     "serverless-http": "^2.3.1",
-    "uuid": "^3.4.0"
+    "uuid": "^3.4.0",
+    "@types/express-serve-static-core": "^4.17.2"
   },
   "devDependencies": {
     "@types/errorhandler": "^0.0.32",

--- a/src/app.ts
+++ b/src/app.ts
@@ -20,7 +20,7 @@ import { applicationErrorMapper, httpErrorHandler, unknownErrorHandler } from '.
 
 const configVersionSupported: ConfigVersion = 1;
 
-export function generateServerlessRouter(fhirConfig: FhirConfig, supportedGenericResources: string[]) {
+export function generateServerlessRouter(fhirConfig: FhirConfig, supportedGenericResources: string[]): any {
     if (configVersionSupported !== fhirConfig.configVersion) {
         throw new Error(`This router does not support ${fhirConfig.configVersion} version`);
     }

--- a/src/app.ts
+++ b/src/app.ts
@@ -3,7 +3,7 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
-import express from 'express';
+import express, { Express } from 'express';
 import {
     cleanAuthHeader,
     getRequestInformation,
@@ -20,7 +20,7 @@ import { applicationErrorMapper, httpErrorHandler, unknownErrorHandler } from '.
 
 const configVersionSupported: ConfigVersion = 1;
 
-export function generateServerlessRouter(fhirConfig: FhirConfig, supportedGenericResources: string[]): any {
+export function generateServerlessRouter(fhirConfig: FhirConfig, supportedGenericResources: string[]): Express {
     if (configVersionSupported !== fhirConfig.configVersion) {
         throw new Error(`This router does not support ${fhirConfig.configVersion} version`);
     }

